vaadin-radio-button-flow-integration-tests/src/test/java/com/vaadin/flow/component/radiobutton/tests/RadioButtonGroupDataViewPageIT.java

```diff
@@ -0,0 +1,243 @@
+
+import java.util.List;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+
+import com.vaadin.flow.testutil.AbstractComponentIT;
+import com.vaadin.flow.testutil.TestPath;
+
+import static com.vaadin.flow.component.radiobutton.tests.RadioButtonGroupDataViewPage.CURRENT_ITEM_SPAN;
+import static com.vaadin.flow.component.radiobutton.tests.RadioButtonGroupDataViewPage.DATA_VIEW_UPDATE_BUTTON;
+import static com.vaadin.flow.component.radiobutton.tests.RadioButtonGroupDataViewPage.HAS_NEXT_ITEM_SPAN;
+import static com.vaadin.flow.component.radiobutton.tests.RadioButtonGroupDataViewPage.HAS_PREV_ITEM_SPAN;
+import static com.vaadin.flow.component.radiobutton.tests.RadioButtonGroupDataViewPage.LIST_DATA_VIEW_ADD_BUTTON;
+import static com.vaadin.flow.component.radiobutton.tests.RadioButtonGroupDataViewPage.LIST_DATA_VIEW_ADD_FILTER_BUTTON;
+import static com.vaadin.flow.component.radiobutton.tests.RadioButtonGroupDataViewPage.LIST_DATA_VIEW_NEXT_BUTTON;
+import static com.vaadin.flow.component.radiobutton.tests.RadioButtonGroupDataViewPage.LIST_DATA_VIEW_PREV_BUTTON;
+import static com.vaadin.flow.component.radiobutton.tests.RadioButtonGroupDataViewPage.LIST_DATA_VIEW_REMOVE_BUTTON;
+import static com.vaadin.flow.component.radiobutton.tests.RadioButtonGroupDataViewPage.LIST_DATA_VIEW_REMOVE_FILTER_BUTTON;
+import static com.vaadin.flow.component.radiobutton.tests.RadioButtonGroupDataViewPage.LIST_DATA_VIEW_SET_FILTER_BUTTON;
+import static com.vaadin.flow.component.radiobutton.tests.RadioButtonGroupDataViewPage.LIST_DATA_VIEW_SORT_BUTTON;
+import static com.vaadin.flow.component.radiobutton.tests.RadioButtonGroupDataViewPage.LIST_DATA_VIEW_UPDATE_BUTTON;
+import static com.vaadin.flow.component.radiobutton.tests.RadioButtonGroupDataViewPage.RADIO_GROUP_FOR_ADD_TO_DATA_VIEW;
+import static com.vaadin.flow.component.radiobutton.tests.RadioButtonGroupDataViewPage.RADIO_GROUP_FOR_DATA_VIEW;
+import static com.vaadin.flow.component.radiobutton.tests.RadioButtonGroupDataViewPage.RADIO_GROUP_FOR_FILTER_DATA_VIEW;
+import static com.vaadin.flow.component.radiobutton.tests.RadioButtonGroupDataViewPage.RADIO_GROUP_FOR_LIST_DATA_VIEW;
+import static com.vaadin.flow.component.radiobutton.tests.RadioButtonGroupDataViewPage.RADIO_GROUP_FOR_REMOVE_FROM_DATA_VIEW;
+import static com.vaadin.flow.component.radiobutton.tests.RadioButtonGroupDataViewPage.RADIO_GROUP_FOR_SORT_DATA_VIEW;
+
+@TestPath("radio-Button-group-data-view")
+public class RadioButtonGroupDataViewPageIT extends AbstractComponentIT {
+
+    private static final String CHANGED_1 = "changed-1";
+    private static final String VAADIN_RADIO_BUTTON = "vaadin-radio-button";
+    private static final String FIRST = "first";
+    private static final String SECOND = "second";
+
+    private static boolean isOpened = false;
+
+    @Before
+    public void openPage() {
+
+        if (isOpened)
```

Comment:
Why do we need it? If you want to open the page only once, maybe use `@BeforeClass` ? Or leave a comment why do you use this flag and condition

Reply:
Yeah I need to open the page only once, but `@BeforeClass` is need to be placed on `static` methods, then you cannot call the `open()` method since it is not static.